### PR TITLE
Reduce verbosity for ray events

### PR DESCRIPTION
**Pull request by richardliaw**
Adds a verbose flag to RayParams to toggle events. Defaults to False, but open to discuss.

The main thing is that when using AIR/Tune, this logging becomes quite verbose.

### CODE REVIEW

File: xgboost_ray/main.py [hunk 1]
```diff
@@ -382,6 +384,8 @@ class RayParams:
     # Distributed callbacks
     distributed_callbacks: Optional[List[DistributedCallback]] = None
 
+    verbose: bool = False
```
Comment (krfricke) [inline xgboost_ray/main.py:387]:
```suggestion
    verbose: bool = True
```

can we default this to True for non-tune runs and to False for tune runs (which includes Ray AIR)?

**Review by krfricke — APPROVED** [review]
Generally lgtm but one question for default

File: xgboost_ray/main.py (hunk 1 above)
Comment (richardliaw) [reply xgboost_ray/main.py:387]: Hmm, how do you do that?

**Comment by Yard1** [comment]
CI should be fixed by https://github.com/ray-project/xgboost_ray/pull/228

**Pull request merged**